### ISSUE TITLE
Provide api to retrieve LSCG keywords for other extensions

### ISSUE THREAD

**Issue by littlesera** [littlesera/LSCG#339]
- drug keywords
- configured spell names
- hypnosis trigger words/phrases?
- configured activity names?

**Comment by bananarama92** [comment]
Not sure where you'd like to expose such an API, but an option would be to make `lscg` namespace globally available. You can expose objects there via an `index.ts` file in combination with an `export` statement.

For example, https://github.com/bananarama92/MBS/blob/cffd0be5aac7bfcbdebfd87b800ece21f213b93b/src/index.ts#L12 currently exposes `mbs.runTests`, `mbs.MBS_VERSION` and `mbs.getFunctionHash`.

**Comment by anchillabc** [comment]
I'd personally love to be able to use the /do command from mbchc with lscg activities like hug etc. so the activity names would be useful

**Comment by littlesera** [comment]
LSCG.DrugKeywords(): string[]
LSCG.NetgunKeywords(): string[]
LSCG.CraftableItemSpellNames(): string[]
LSCG.HypnoTriggers(): string[]
LSCG.ConfiguredActivities(): ActivityEntryModel[]

**Comment by bananarama92** [comment]
Thanks for the API Sera! Got a question though: where exactly is this `LSCG` namespace located? I'm noticing that it's not a part of the global namespace.

**Comment by littlesera** [comment]
Hmm, ought to be a global namespace:
![image](https://github.com/littlesera/LSCG/assets/126942824/dd32937b-fa56-49a1-bd8b-e24da5805f51)

**Comment by bananarama92** [comment]
Odd, I'm getting a good old reference error with both LSCG v0.3.17 and v0.3.16 (loaded via FUSAM).

```js
>> LSCG
Uncaught ReferenceError: LSCG is not defined
    <anonymous> debugger eval code:1
```

**Comment by littlesera** [comment]
Fixed in #366 
PR submitted for FUStAM: https://gitlab.com/Sidiousious/bc-addon-loader/-/merge_requests/7
Tampermonkey add will need updating.
